things/bbcr-ar.cdb

```diff
@@ -0,0 +1,10 @@
+title BBCR
+type registry
+appeared 1993
+standsFor Bahía Blanca Cancer Registry 
+description The Regional Tumor Registry of the South of the Province of Buenos Aires covers the political area of Bahía Blanca, with a population of 290 000 at the 2005 census.Although the registry was recognized by a provincial law and a municipal resolution in 1993, it does not have a speciĺc budget. The Argentine Association for Cancer Prevention and Education (AAPEC), a non-profit organization founded in 19 to improve cancer control, maintains the registry and other high-priority projects, such as the tobacco control programme. Although resources are limited, the registry employs two registrars and a registry coordinator. Most of the registry’s resources are provided by the municipal government of Bahía Blanca.
```

Comment:
Can you add an entry for AAPEC?

Reply:
@breck7 There are multiple AAPEC's. I don't know which/what you are referring to